plugin-modernizer-core/src/test/java/io/jenkins/tools/pluginmodernizer/core/recipes/DeclarativeRecipesTest.java

```diff
@@ -1396,6 +1463,7 @@ void upgradeNextMajorParentVersionTest() {
                                 <connection>scm:git:https://github.com/jenkinsci/empty-plugin.git</connection>
                               </scm>
                               <properties>
+                                <jenkins-test-harness.version>RELEASE</jenkins-test-harness.version>
```

Comment:
https://docs.openrewrite.org/reference/dependency-version-selectors#supported-dependency-version-selectors

By using latest.release you will get the latest version